pkgs/development/node-packages/node-packages.nix

```diff
@@ -81407,66 +81344,6 @@ in
     bypassCache = true;
     reconstructLock = true;
   };
-  prisma = nodeEnv.buildNodePackage {
-    name = "prisma";
-    packageName = "prisma";
-    version = "5.17.0";
-    src = fetchurl {
-      url = "https://registry.npmjs.org/prisma/-/prisma-5.17.0.tgz";
-      sha512 = "m4UWkN5lBE6yevqeOxEvmepnL5cNPEjzMw2IqDB59AcEV6w7D8vGljDLd1gPFH+W6gUxw9x7/RmN5dCS/WTPxA==";
-    };
-    dependencies = [
-      sources."@prisma/debug-5.17.0"
-      sources."@prisma/engines-5.17.0"
-      sources."@prisma/engines-version-5.17.0-31.393aa359c9ad4a4bb28630fb5613f9c281cde053"
-      sources."@prisma/fetch-engine-5.17.0"
-      sources."@prisma/get-platform-5.17.0"
-    ];
-    buildInputs = globalBuildInputs;
-    meta = {
-      description = "Prisma is an open-source database toolkit. It includes a JavaScript/TypeScript ORM for Node.js, migrations and a modern GUI to view and edit the data in your database. You can use Prisma in new projects or add it to an existing one.";
-      homepage = "https://www.prisma.io";
-      license = "Apache-2.0";
-    };
-    production = true;
-    bypassCache = true;
-    reconstructLock = true;
-  };
-  "@prisma/language-server" = nodeEnv.buildNodePackage {
-    name = "_at_prisma_slash_language-server";
-    packageName = "@prisma/language-server";
-    version = "5.17.0";
-    src = fetchurl {
-      url = "https://registry.npmjs.org/@prisma/language-server/-/language-server-5.17.0.tgz";
-      sha512 = "4XXTBGpwJBUFqvY2WxL/4W1q/OOstxCJER4TYftNgMGjKjnsFkQNme567QTlPwWp7VGw2jcNmyTFOcP4HdohIg==";
-    };
-    dependencies = [
-      sources."@prisma/prisma-schema-wasm-5.17.0-31.393aa359c9ad4a4bb28630fb5613f9c281cde053"
-      sources."@prisma/schema-files-loader-5.17.0"
-      sources."@types/js-levenshtein-1.1.3"
-      sources."fs-extra-11.1.1"
-      sources."graceful-fs-4.2.11"
-      sources."js-levenshtein-1.1.6"
-      sources."jsonfile-6.1.0"
-      sources."klona-2.0.6"
-      sources."universalify-2.0.1"
-      sources."vscode-jsonrpc-8.1.0"
-      sources."vscode-languageserver-8.1.0"
-      sources."vscode-languageserver-protocol-3.17.3"
-      sources."vscode-languageserver-textdocument-1.0.11"
-      sources."vscode-languageserver-types-3.17.3"
-      sources."vscode-uri-3.0.8"
-    ];
-    buildInputs = globalBuildInputs;
-    meta = {
-      description = "Prisma Language Server";
-      homepage = "https://github.com/prisma/language-tools#readme";
-      license = "Apache-2.0";
-    };
-    production = true;
-    bypassCache = true;
-    reconstructLock = true;
-  };
```

Comment:
I don't know how to re-generate this file without touching unrelated packages. I manually deleted all the `prisma` references, but I'm sure there are probably some zombie dependencies left as a result. Let me know if there's a better way to do this.